package.json

```diff
@@ -138,33 +146,94 @@
+   
+   "build:watch:input": "nodemon -e scss,js --watch \"./components/input/src\" --ignore \"*.css\" --ignore \"*-css.js\" --ignore \"*.version.js\" --ignore \"*Version.js\" --exec \"npm run build:input\"",
+   "build:watch:dropdown": "nodemon -e scss,js --watch \"./components/dropdown/src\" --ignore \"*.css\" --ignore \"*-css.js\" --ignore \"*.version.js\" --ignore \"*Version.js\" --exec \"npm run build:dropdown\"",
+   
+   "dist:js:input": "copyfiles -u 1 -V \"./components/input/src/**/*.js\" ./dist",
+   "dist:js:dropdown": "copyfiles -u 1 -V \"./components/dropdown/src/**/*.js\" ./dist",
+   
+   "postCss:input": "node ./node_modules/@aurodesignsystem/auro-library/scripts/build/postCss.mjs",
+   "postCss:dropdown": "node ./node_modules/@aurodesignsystem/auro-library/scripts/build/postCss.mjs",
+   
+   "sass:render": "sass-render --load-path=node_modules \"components/**/*.css\" -t ./node_modules/@aurodesignsystem/auro-library/scripts/build/staticStyles-template.js",
+   "sass:render:input": "sass-render --load-path=node_modules \"./components/input/src/**/*.css\" -t ./node_modules/@aurodesignsystem/auro-library/scripts/build/staticStyles-template.js",
+   "sass:render:dropdown": "sass-render --load-path=node_modules \"./components/dropdown/src/**/*.css\" -t ./node_modules/@aurodesignsystem/auro-library/scripts/build/staticStyles-template.js",
+   
+   "serve": "web-dev-server --open demo/ --node-resolve --watch",
+   "serve:input": "web-dev-server --open components/input/demo/ --node-resolve --watch --app-index \"./components/input/demo/index.html\"",
+   "serve:dropdown": "web-dev-server --open components/dropdown/demo/ --node-resolve --watch --app-index \"./components/dropdown/demo/index.html\"",
+
+   "build:input:full": "npm-run-all build:input build:api:input build:docs:input build:dev:assets:input",
+   "build:dropdown:full": "npm-run-all build:dropdown build:api:dropdown build:docs:dropdown build:dev:assets:dropdown",
+   
+   "build:input:release": "npm-run-all build:input:full build:test build:demo:input",
+   "build:dropdown:release": "npm-run-all build:dropdown:full build:test build:demo:dropdown",
+
+   "sweep": "rm -rf dist && find ./demo ./src ./components -not -path \"*/node_modules/*\" -type f \\( -name \"*.css\" -o -name \"*-css.js\" \\) -delete",
+
+   "test": "wtr --coverage",
+    "test:watch": "wtr --watch",
+    "test:input": "wtr \"./components/input/test/**/*.test.js\" --coverage",
+    "test:input:watch": "wtr \"./components/input/test/**/*.test.js\" --watch",
+    "test:dropdown": "wtr \"./components/dropdown/test/**/*.test.js\" --coverage",
+    "test:dropdown:watch": "wtr \"./components/dropdown/test/**/*.test.js\" --watch",
+
+    "linters": "npm-run-all scssLint esLint",
+    "scssLint": "stylelint \"./components/**/*.scss\"",
+    "esLint": "./node_modules/.bin/eslint components/**/*.js --ignore-pattern 'components/**/*Version.js'",
+
+    "linters:input": "npm-run-all scssLint:input esLint:input",
```

Comment:
this failed with exit-code 2

Reply:
Updated here: https://github.com/AlaskaAirlines/auro-form/pull/16/commits/8c56ee54efaf479bded11236fc9d91b44990148e

Linting as expected:

```
 $ npm run linters:input

> @aurodesignsystem/auro-form@0.0.0 linters:input
> npm-run-all scssLint:input esLint:input

> @aurodesignsystem/auro-form@0.0.0 scssLint:input
> stylelint "./components/input/**/*.scss"

> @aurodesignsystem/auro-form@0.0.0 esLint:input
> eslint 'components/input/**/*.js'
```